docs/getting-started/installation.md

```diff
@@ -109,14 +109,17 @@ Compose is currently published in two separate variants:
 
 === "Since v3.0.0-alpha01"
 
-    All Compose variants (Android, JVM/Desktop, Native/iOS, Native/macOS, JS/Canvas, Wasm) are published from the main branch, please add dependencies as usual (no version suffixes required).
+    All Compose variants (Android, JVM/Desktop, Native/iOS, Native/macOS, JS/Canvas, Wasm) are published from the main branch,
+    please add dependencies as usual (no version suffixes required).
```

Comment:
Unnecessary line wrap.

Reply:
This is also done. Sorry for marking this earlier as resolved.